src/components/FeatureViews/AdminPortal/AdminPortalTabs.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import { makeStyles } from '@material-ui/core/styles';
+import BusinessDetails from './BusinessDetails';
+
+const useStyles = makeStyles({
+    adminPortalTabsStyle: {
+        marginTop: '2rem',
+    },
+});
+
+const AdminPortalTabs = () => {
+    const [tabIndex, setTabIndex] = useState(0);
+    const handleTabChange = (event, currentTabIndex) => {
+        setTabIndex(currentTabIndex);
+    };
+
+    const classes = useStyles();
+
+    const adminPortalTabs = [
+        { label: 'overview', tab_id: 0 },
```

Comment:
isn't the id just the index? and i don't think we're using the tab_index so may want to consider deleting?